src/routes/wohnen/components/Offer.js

```diff
@@ -0,0 +1,137 @@
+      const accommodation: AccommodationType = specificOffer.formData.accommodation
+      const costs = specificOffer.formData.costs
+      const landlord = offer.formData.landlord
+
+      const translateRunningServices = keys => this.stringify(this.translate('runningServices', keys))
+      const translateAdditionalServices = keys => this.stringify(this.translate('additionalServices', keys))
+      const translateRooms = keys => this.stringify(this.translate('rooms', keys))
+
+      return <React.Fragment>
+        <div>
+          <ListElement>
+            <Header>Mietobjekt</Header>
+          </ListElement>
+          <Row>
+            <RowTitle>Gesamtfläche:</RowTitle>
+            <RowValue> {accommodation.totalArea} m²</RowValue>
+          </Row>
+          <Row>
+            <RowTitle>Zimmeranzahl:</RowTitle>
+            <RowValue> {accommodation.totalRooms}</RowValue>
+          </Row>
+          <Row>
+            <RowTitle>Zimmer:</RowTitle>
+            <RowValue> {translateRooms(accommodation.ofRooms)}</RowValue>
+          </Row>
+        </div>
+
+        <div>
+          <ListElement>
+            <Header>Mietkosten</Header>
+          </ListElement>
+          <Row>
+            <RowTitle>Grundmiete:</RowTitle>
+            <RowValue> {costs.baseRent} € monatlich</RowValue>
+          </Row>
+          <Row>
+            <RowTitle>Nebenkosten:</RowTitle>
+            <RowValue> {costs.runningCosts} € monatlich</RowValue>
+          </Row>
+          <Row>
+            <RowTitle>In Nebenkosten enthalten:</RowTitle>
+            <RowValue> {translateRunningServices(costs.ofRunningServices)}</RowValue>
+          </Row>
+          <Row>
+            <RowTitle>Nicht in Nebenkosten enthalten:</RowTitle>
+            <RowValue> {translateRunningServices(costs.ofRunningServicesDiff)}</RowValue>
+          </Row>
+          <Row>
+            <RowTitle>Warmwasser in Nebenkosten?</RowTitle>
+            <RowValue> {costs.hotWaterInHeatingCosts ? 'Ja' : 'Nein'}</RowValue>
+          </Row>
+          <Row>
+            <RowTitle>Zusatzkosten:</RowTitle>
+            <RowValue> {costs.additionalCosts} € monatlich</RowValue>
+          </Row>
+          <Row>
+            <RowTitle>In Zusatzkosten enthalten:</RowTitle>
+            <RowValue> {translateAdditionalServices(costs.ofAdditionalServices)}</RowValue>
+          </Row>
+          <Row>
+            <RowTitle>Nicht in Zusatzkosten enthalten:</RowTitle>
+            <RowValue> {translateAdditionalServices(costs.ofAdditionalServicesDiff)}</RowValue>
+          </Row>
+        </div>
+
+        <div>
+          <ListElement>
+            <Header>Kontaktdaten</Header>
+          </ListElement>
+          <Row>
+            <RowTitle>Name:</RowTitle>
+            <RowValue> {landlord.firstName} {landlord.lastName}</RowValue>
+          </Row>
+          <Row>
+            <RowTitle>Email:</RowTitle>
+            <RowValue> {offer.email}</RowValue>
```

Comment:
Maybe you could make this an href with `mailto:xyz@email.com` (maybe we need to pay attention to escaping the string)